code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1292,23 +1291,25 @@
 	. = ..()
 	if(. & EMP_PROTECT_WIRES || !IS_ROBOTIC_LIMB(src))
 		return FALSE
-	owner.visible_message(span_danger("[owner]'s [src.name] seems to malfunction!"))
 
 	// with defines at the time of writing, this is 3 brute and 2 burn
 	// 3 + 2 = 5, with 6 limbs thats 30, on a heavy 60
 	// 60 * 0.8 = 48
-	var/time_needed = 10 SECONDS
+	var/time_needed = 0
 	var/brute_damage = AUGGED_LIMB_EMP_BRUTE_DAMAGE
 	var/burn_damage = AUGGED_LIMB_EMP_BURN_DAMAGE
 	if(severity == EMP_HEAVY)
-		time_needed *= 2
+		time_needed = AUGGED_LIMB_EMP_PARALYZE_TIME
```

Comment:
This doesn't match your PR body which states that it's 3 seconds light / 6 seconds heavy, only if damaged.
The actual effect seems to be 3 seconds, heavy emp only, only if already damaged.
I like the PR described version better than this one.

Reply:
I think losing control of your legs for 3 seconds when emped is not only reasonable but what you would expect to happen if you had robotic legs.